schema.go

```diff
@@ -297,6 +297,7 @@ func structNodeOf(t reflect.Type) *structNode {
 
 func structFieldsOf(t reflect.Type) []reflect.StructField {
```

Comment:
I'm not familiar with structure fields/tags yet so not quite certain this is the place for this kind of change. Tests show that it is working as intended but would like to get some feedback on this.

Reply:
I think `appendStructFields` is the bottom-most function collecting struct fields.

What if we just don't append the fields named `-` in `appendStructFields`, would that work?

Reply:
better :) thanks